Integrations/integration-Lastline.yml

```diff
@@ -612,6 +639,18 @@ script:
                 }
                 return entries;
 
+            case 'lastline-check-status':
+                var uuids = argToList(args.uuid);
+                var entries = [];
+                for (var i=0; i < uuids.length; i++) {
+                    var reportResponse = CheckStatus(uuids[i]);
+                    if (reportResponse.error){
```

Comment:
you don't have an error in the response, it's either a string or an entry dictionary.
maybe you need try-catch?

Reply:
i think it's not needed. since i catch the report not ready error and return it as a string. other errors should stop the execution.